OPT_mission.Tanoa/sectorcontrol/functions/fnc_postinit.sqf

```diff
@@ -37,7 +37,7 @@ if (hasInterface) then {
         ["Flagge ziehen", "Flagge ziehen, wenn anvisiert."], 
         {
             if (
-                cursorObject in (GVARMAIN(nato_flags) + GVARMAIN(csat_flags)) and
+                [cursorObject] in (GVARMAIN(nato_flags) + GVARMAIN(csat_flags)) and
```

Comment:
Wenn die Flaggen nicht in extra-Arrays verpackt werden, kann hier auch `cursorObject` ohne extra Verpackung verwendet werden

Reply:
Selbe Problematik mit Einzelnen/Mehreren Flaggen wie oben.